riff-raff/build.sbt

```diff
@@ -6,13 +6,15 @@ resolvers ++= Seq(
     "Typesafe Releases" at "http://repo.typesafe.com/typesafe/releases/"
 )
 
+ivyScala := ivyScala.value map { _.copy(overrideScalaVersion = true) }
```

Comment:
Ah - that does seem to work. So it's only when you pull in a library that is using a newer scala than your build. That makes it eminently fixable. It's not a great warning message. I avoided adding it for ages, but then I saw [others were using it](https://github.com/search?utf8=%E2%9C%93&q=ivyScala+user%3Aguardian&type=Code&ref=searchresults) and figured I'd join them (just realised that most of those are my own team).